torch/_refs/nn/functional/__init__.py

```diff
@@ -233,6 +237,46 @@ def selu(a: TensorLikeType, inplace: bool = False) -> TensorLikeType:
     return scale * torch.where(a > 0, a, rhs)
 
 
+# copy from torch.nn.functional
+def _get_softmax_dim(name: str, ndim: int, stacklevel: int) -> int:
+    warnings.warn(
+        "Implicit dimension choice for {} has been deprecated. "
+        "Change the call to include dim=X as an argument.".format(name),
+        stacklevel=stacklevel,
+    )
```

Comment:
We should probably just error out in these cases that have been deprecated.

Reply:
Also, we should remove the `_stacklevel` arg, as it's a private argument.

Reply:
so, just remove the underscore arg from the refs, correct?

Reply:
yep, and the warning and all that.

Reply:
In general, it's fine for refs not to implement behaviour that's been deprecated. We want refs to be forward-looking.